src/main/java/redis/clients/jedis/search/aggr/FtAggregateIteration.java

```diff
@@ -1,25 +1,33 @@
 package redis.clients.jedis.search.aggr;
 
+import java.util.Collection;
+
 import redis.clients.jedis.CommandArguments;
 import redis.clients.jedis.providers.ConnectionProvider;
 import redis.clients.jedis.search.SearchBuilderFactory;
 import redis.clients.jedis.search.SearchProtocol;
-import redis.clients.jedis.util.JedisRoundRobinBase;
+import redis.clients.jedis.util.JedisCommandIterationBase;
 
-public class FtAggregateRoundRobin extends JedisRoundRobinBase<AggregationResult> {
+public class FtAggregateIteration extends JedisCommandIterationBase<AggregationResult, Row> {
```

Comment:
@bsbodden @maorohana-redis Also let me know whether you prefer `Row` or `Map<String, Object>` for aggregation replies.

Reply:
I like the concept of `Row` since you could also pack any metadata about its results. Like in VSS for example, the score could come as part of `Row` but let's hear from Maor too.